src/components/todo/TodoForm.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components';
+import { Button } from '../Button';
+import { useState } from 'react';
+
+// 할 일 입력 폼
+export default function TodoForm({ setTodos }) {
+  const [inputValue, setInputValue] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(inputValue);
+    if (inputValue.trim()) {
+      setTodos((prevTodos) => [
+        ...prevTodos,
+        {
+          id: Date.now(), // 유니크한 ID 생성
+          value: inputValue,
+          isCompleted: false,
+        },
+      ]);
+      setInputValue(''); // 입력 필드 초기화
+    }
+  };
+
+  return (
+    <StyledForm onSubmit={handleSubmit}>
+      <input
+        type="text"
+        placeholder="할 일 추가"
+        value={inputValue} // inputValue 상태값을 value 속성에 연결 : 값 가져오려면 필요함
+        onChange={(e) => setInputValue(e.target.value)}
+      />
+      <Button
+        type="submit"
+        onClick={handleSubmit}
+      >
+        추가
+      </Button>
```

Comment:
handleSubmit 함수에서 버튼의 onClick 핸들러 중복됩니다!

Form에서는 아래와 같이 작성해도 정상 작동해요

```suggestion
<Button type="submit">
  추가
</Button>

```

Reply:
오 감사합니다!